vcfg rmq catch the missing directory and show a presentable error

When running the following one should just get a missing rmq server download tell the user what to do to get it...rather than throwing the trace below.

(volttron) (base) osboxes@ubuntu2004:~/repos/volttron$ vcfgYour VOLTTRON_HOME currently set to: /home/osboxes/.volttronIs this the volttron you are attempting to setup? [Y]: 
What type of message bus (rmq/zmq)? [zmq]: rmq

// code placeholderWhat type of message bus (rmq/zmq)? [zmq]: rmq
Traceback (most recent call last):
 File "/home/osboxes/repos/volttron/env/bin/vcfg", line 11, in <module>
 load_entry_point('volttron', 'console_scripts', 'vcfg')()
 File "/home/osboxes/repos/volttron/volttron/platform/config.py", line 593, in _main
 main()
 File "/home/osboxes/repos/volttron/volttron/platform/instance_setup.py", line 1017, in main
 wizard()
 File "/home/osboxes/repos/volttron/volttron/platform/instance_setup.py", line 864, in wizard
 do_message_bus()
 File "/home/osboxes/repos/volttron/volttron/platform/instance_setup.py", line 426, in do_message_bus
 check_rmq_setup()
 File "/home/osboxes/repos/volttron/volttron/platform/instance_setup.py", line 394, in check_rmq_setup
 rmq_config = RMQConfig()
 File "/home/osboxes/repos/volttron/volttron/utils/rmq_config_params.py", line 72, in __init__
 assert os.path.isdir(self.rabbitmq_server), "Missing rabbitmq server directory{}".format(self.rabbitmq_server)
AssertionError: Missing rabbitmq server directory/home/osboxes/rabbitmq_server/rabbitmq_server-3.7.7

--- a/volttron/platform/instance_setup.py
+++ b/volttron/platform/instance_setup.py
@@ -423,7 +423,12 @@ def do_message_bus():
         #     print("Rabbitmq dependencies not installed. Installing now...")
         #     set_dependencies("rabbitmq")
         #     print("Done!")
-        check_rmq_setup()
+        try:
+            check_rmq_setup()
+        except AssertionError:
+            print("RabbitMQ setup is incomplete. RabbitMQ server directory is missing.")
+            print("Please run bootstrap --rabbitmq before running vcfg")
+            sys.exit()
 
     config_opts['message-bus'] = bus_type
 

